docker/pytorch/Dockerfile

```diff
@@ -184,3 +184,19 @@ RUN sudo pip${PYTHON_VERSION} install --no-cache-dir \
     cupy-`echo ${CUDA_VERSION_TAG} | sed "s/cu/cuda/g"`>=10.2.0 && \
     sudo pip${PYTHON_VERSION} install --no-cache-dir mmcv-full==1.4.4 -f https://download.openmmlab.com/mmcv/dist/${CUDA_VERSION_TAG}/${MMCV_TORCH_VERSION}/index.html && \
     sudo pip${PYTHON_VERSION} install --no-cache-dir 'mmsegmentation>=0.22.0,<0.23'
+
+
+FROM pytorch_stage AS mofed_stage
+ARG DEBIAN_FRONTEND=noninteractive
+
+###################################
+# Mellanox OFED driver installation
```

Comment:
Will we need a vision  + mellanox combo image?

Reply:
Probably not yet - I'm fine with only adding that when someone requests it. `pytorch_vision` isn't even public in Dockerhub yet, though it probably should be?